lib/logstash/inputs/beats_support/connection_handler.rb

```diff
@@ -40,7 +43,11 @@ def process(hash, identity_stream)
                                       :peer => @connection.peer)
 
       # Filebeats uses the `message` key and LSF `line`
-      target_field = @input.target_field_for_codec ? hash.delete(@input.target_field_for_codec) : nil
+      target_field = if from_filebeat?(hash)
+                       hash.delete(FILEBEAT_LOG_LINE_FIELD)
+                     elsif from_logstash_forwarder?(hash)
+                       hash.delete(LSF_LOG_LINE_FIELD)
```

Comment:
The other if will take care of it, it means we have received data from something other than filebeat itself.. Could be `topbeat`